package_manifest.yaml

```diff
@@ -492,8 +492,8 @@ foreman_client_packages:
       el10: "{{ hostvars['foreman-client-staging-repoclosure-el10']['repoclosure_lookaside_repos']['el10'] }}"
       el9: "{{ hostvars['foreman-client-staging-repoclosure-el9']['repoclosure_lookaside_repos']['el9'] }}"
       el8: "{{ hostvars['foreman-client-staging-repoclosure-el8']['repoclosure_lookaside_repos']['el8'] }}"
-      leap155:
-        - leap155-oss
+      leap156:
+        - leap156-oss
         - atix-sles155
```

Comment:
I wasn't sure what modifications, if any, were needed to the atix-sles155 repo, so I left it untouched (for now).

there _is_ a SLES15SP6 repo that is available, but I figured it would be a simple fixup after this if needed.

Reply:
I'd flip it to sp6 too, is anything else then for being consistent.